src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CertificateStore.java

```diff
@@ -142,15 +141,24 @@ public PrivateKey getCAPrivateKey() {
      * @throws KeyStoreException if unable to retrieve the certificate
      */
     public X509Certificate getCACertificate() throws KeyStoreException {
```

Comment:
```suggestion
    public X509Certificate getCACertificate() {
```
same w/  javadoc

Reply:
I actually went the other direction. Downstream consumers are catching KeyStoreException and then passing straight through to methods that are annotated with @NotNull.
Maybe there's something we can do in the future to prevent this from ever getting into a bad state. But for now, this is a more surgical change.